src/index.js

```diff
@@ -0,0 +1,137 @@
+
+const COLORS = ["green","red","orange","yellow","teal"]
+const changeColor = (temperature) =>{
+    const element = document.getElementById("temperatureSection");
+    if(temperature >= 80){
+        element.classList.forEach((className) => COLORS.includes(className) ? element.classList.remove(className) : className )
+        element.classList.add('red');
+        landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    }
+    else if(temperature >= 70 && temperature <=79){
+        element.classList.forEach((className) => COLORS.includes(className) ? element.classList.remove(className) : className )
+        element.classList.add('orange');
+        landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    }
+    else if(temperature >= 60 && temperature <=69){
+        element.classList.forEach((className) => COLORS.includes(className) ? element.classList.remove(className) : className )
+        element.classList.add('yellow');
+        landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    }
+    else if(temperature >= 50 && temperature <=59){
+        element.classList.forEach((className) => COLORS.includes(className) ? element.classList.remove(className) : className )
+        element.classList.add('green');
+        landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
+    else {
+        element.classList.forEach((className) => COLORS.includes(className) ? element.classList.remove(className) : className )
+        element.classList.add('teal');
+        landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
```

Comment:
When we have large if-statements like this where each if-block performs the same operations with slightly different values, it can be worth making a helper function that handles the updates to reduce places we're repeating code and could accumulate small typos. With the helper function, each of the ifs here would call that function passing the desired parameters rather than setting attributes themselves.